clientd3d/annotate.h

```diff
@@ -12,9 +12,9 @@
 #ifndef _ANNOTATE_H
 #define _ANNOTATE_H
 
-#define MAP_ANNOTATION_SIZE     (2 * FINENESS)    // Size of annotation indicator in FINENESS units
-#define MAP_ANNOTATION_MIN_SIZE 14                // Min size of drawn annotation in pixels
-
+#define MAP_ANNOTATION_SIZE       FINENESS          // Size of annotation graphic in FINENESS units
```

Comment:
Confused here--what is "size of annotation graphic"?

Reply:
> the comment could be clearer about this being the size of the source artwork

Maybe "graphic" is the wrong word here? Artwork? Asset? Something else entirely?

Reply:
Oh weird, this is the size of the bitmap on disk?  As in, if that artwork were resized, it wouldn't draw correctly?